mlflow/server/js/src/reducers/Reducers.js

```diff
@@ -203,6 +213,47 @@ const amendTagsByRunUuid = (state, tags, runUuid) => {
   return newState;
 };
 
+const experimentTagsByExperimentId = (state = {}, action) => {
+  const tagArrToObject = (tags) => {
+    const tagObj = {};
+    tags.forEach((tag) => tagObj[tag.key] = ExperimentTag.fromJs(tag));
+    return tagObj;
+  };
+  switch (action.type) {
+    case fulfilled(GET_EXPERIMENT_API): {
+      const {experiment} = action.payload;
```

Comment:
nit: space after the brace.

Reply:
You may want to try this formatter: https://tinyurl.com/y2wjtxlz

Reply:
Thanks for the tip! Updated the formatting.